object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+const pick = (source, keys) => {
+
+  const obj = {};
+
+  for (const property in source) {
+    for (let i = 0; i < keys.length; i++) {
+      if (property === keys[i]) {
+        obj[keys[i]] = source[keys[i]];
+      }
+    }
+  }
```

Comment:
This is pretty good, see if you can apply similar strategies to some of your other solutions.